lib/components/base/Pagination.vue

```diff
@@ -15,7 +15,7 @@
       :key="'page-' + item + '-' + index"
       :class="{
         'page-number': page !== item,
-        shrink: item > 99,
+        shrink: (item as number) > 99,
```

Comment:
```suggestion
        shrink: item !== '-' && item > 99,
```

Given that you narrowed the type of `pages` ref (according to the comment below), instead of casting to a number here, just check that the item is not `'-'`.